packages/core/src/accordion/accordion.stories.mdx

```diff
@@ -0,0 +1,51 @@
+import { Meta, Props, Story, Preview, API } from '@storybook/addon-docs/blocks';
+import { html, LitElement } from 'lit-element';
+
+<Meta title="Components/Accordion/Getting Started" />
```

Comment:
The stories for the accordion need to go into the components preview section. New components go through one full version preview.